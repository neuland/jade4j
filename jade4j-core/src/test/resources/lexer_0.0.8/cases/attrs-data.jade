- var user = { name: 'tobi' }
foo(data-user=user)
foo(data-items=[1,2,3])
foo(data-username='tobi')
foo(data-escaped={message: "Let's rock!"})
