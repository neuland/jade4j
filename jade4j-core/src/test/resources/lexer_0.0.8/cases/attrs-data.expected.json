{"type":"code","line":1,"val":"var user = { name: 'tobi' }","escape":false,"buffer":false}
{"type":"newline","line":2}
{"type":"tag","line":2,"val":"foo","selfClosing":false}
{"type":"attrs","line":2,"attrs":[{"name":"data-user","val":"user","escaped":true}]}
{"type":"newline","line":3}
{"type":"tag","line":3,"val":"foo","selfClosing":false}
{"type":"attrs","line":3,"attrs":[{"name":"data-items","val":"[1,2,3]","escaped":true}]}
{"type":"newline","line":4}
{"type":"tag","line":4,"val":"foo","selfClosing":false}
{"type":"attrs","line":4,"attrs":[{"name":"data-username","val":"'tobi'","escaped":true}]}
{"type":"newline","line":5}
{"type":"tag","line":5,"val":"foo","selfClosing":false}
{"type":"attrs","line":5,"attrs":[{"name":"data-escaped","val":"{message: \"Let's rock!\"}","escaped":true}]}
{"type":"newline","line":6}
{"type":"eos","line":6}